interfaces/cython/cantera/yaml2ck.py

```diff
@@ -0,0 +1,851 @@
+# This file is part of Cantera...
```

Comment:
This should be
```
# This file is part of Cantera. See License.txt in the top-level directory or
# at https://cantera.org/license.txt for license and copyright information.
```
(see also `ck2yaml.py` for comparison: that file includes some other information that may need to be included)